server.go

```diff
@@ -142,8 +142,8 @@ func newServer(listenAddrs []string, chanDB *channeldb.DB, cc *chainControl,
 		outboundPeers:          make(map[string]*peer),
 		peerConnectedListeners: make(map[string][]chan<- struct{}),
 
-		globalFeatures: globalFeatures,
-		localFeatures:  localFeatures,
+		globalFeatures: lnwire.NewGlobalFeatureVector(),
+		localFeatures:  lnwire.NewLocalFeatureVector(lnwire.InitialRoutingSync),
```

Comment:
With this setting the _all_ created peers will _always_ advertise the "sync graph" bit. This type if initialization makes sense for the global features, but in many cases the local features will be context specific. As a result, I think the `newPeer` constructor should be modified to take in the set of local features. With this small change, we'll be able to programmatically determine _which_ bits should be set when creating the peer struct within the server.

Reply:
If you don't mind, I think we can leave this for a future PR. This keeps the currently implemented behavior.